src/configuration.ts

```
@@ -8,13 +8,150 @@ import { Manifest, manifestSchema } from "./types/manifest";
+  octokitByLocation: Map<string, Context["octokit"] | null>;
+};
+
+const ENVIRONMENT_TO_CONFIG_SUFFIX: Record<string, string> = {
+  development: "dev",
+};
+
+const VALID_CONFIG_SUFFIX = /^[a-z0-9][a-z0-9_-]*$/i;
+const MAX_IMPORT_DEPTH = 6;
+
+// eslint-disable-next-line @ubiquity-os/no-empty-strings
+const EMPTY_STRING = "";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'EMPTY_STRING'. Consider to delete it or rename it.

'EMPTY_STRING' is defined here:


(lint/suspicious/noRedeclare)